README.md

```diff
@@ -1,2 +1,24 @@
 # fluent-siren-client
 Fluent client for Siren Hyper Media APIs
+
+Fetches a [Siren](https://github.com/kevinswiber/siren) object using the provided href and request function, and parses 
+it into a [node-siren-parser](https://github.com/Brightspace/node-siren-parser) Entity. 
+Configures the Entity in such a way that uri links can be followed, and actions can be performed. Result of 
+following a link is another fluent Entity.
+
+## Installation
+
+Install from NPM:
+```shell
+npm install fluent-siren-client
+```
+
+## Usage
+
+```javascript
+const client = new SirenClient(requestFn);
```

Comment:
we should specify what the signature of `requestFn()`
maybe right beside this in a comment?
```
const client = new SirenClient(requestFn); // async requestFn(uri, method, fieldValues)
```

or something like that
I had mentioned before that a section below `Usage` for setup would be appropriate.
You could show maybe a really simple way to setup requestFn with just request-promise, and maybe suggest this is where you could do authorization steps if you have any.

Reply:
`As of Feb 11th 2020, request is fully deprecated`. Should we look into different library for implementing `requestFn`, or is it good enough to include an example using the deprecated request-promise?

Reply:
lets just use request promise in the example for now - people can substitute their own lib, it's just an example.
Request is still in maintenance mode.